admin/services/pupils-not-taking-check.service.js

```diff
@@ -1,30 +1,45 @@
 /** @namespace */
 
-const pupilsNotTakingCheckService = {
+const pupilIdentificationFlag = require('../services/pupil-identification-flag.service')
+const pupilDataService = require('./data-access/pupil.data.service')
+const pupilsNotTakingCheckDataService = require('../services/data-access/pupils-not-taking-check.data.service')
+
+const pupilsNotTakingCheckService = {}
 
   /**
    * Sort columns by reason asc/desc.
    * @param pupilsList
    * @param sortDirection
    * @returns {*}
    */
-  sortPupilsByReason: (pupilsList, sortDirection) => {
-    let sortedPupilsList
-    sortedPupilsList = pupilsList.sort((a, b) => {
-      if (a.reason === 'N/A') {
-        return 1
-      } else if (b.reason === 'N/A') {
-        return -1
-      } else if (a.reason === b.reason) {
-        return 0
-      } else if (sortDirection === 'asc') {
-        return a.reason < b.reason ? -1 : 1
-      } else {
-        return a.reason < b.reason ? 1 : -1
-      }
-    })
-    return sortedPupilsList
-  }
+pupilsNotTakingCheckService.sortPupilsByReason = (pupilsList, sortDirection) => {
+  let sortedPupilsList
+  sortedPupilsList = pupilsList.sort((a, b) => {
+    if (a.reason === 'N/A') {
+      return 1
+    } else if (b.reason === 'N/A') {
+      return -1
+    } else if (a.reason === b.reason) {
+      return 0
+    } else if (sortDirection === 'asc') {
+      return a.reason < b.reason ? -1 : 1
+    } else {
+      return a.reason < b.reason ? 1 : -1
+    }
+  })
+  return sortedPupilsList
+}
+
+pupilsNotTakingCheckService.pupils = async (schoolId, sortField, sortDirection) => {
```

Comment:
this method name give me no idea what it does

Reply:
@GuyHarwood I struggled with naming these two methods (see your other comment), I'm relying on the service name to know what it is about. Would `getPupils` be good enough?

Reply:
build the name through its purpose, and prefix it with get.

what is it doing exactly, sorting?